test/integration-tests/IntegrationTests.Helpers/TestHelper.cs

```diff
@@ -59,7 +59,7 @@ public async Task<Container> StartContainerAsync(int traceAgentPort, int webPort
 
         string agentBaseUrl = $"http://{DockerNetworkHelper.IntegrationTestsGateway}:{traceAgentPort}";
         string agentHealthzUrl = $"{agentBaseUrl}/healthz";
-        string zipkinEndpoint = $"{agentBaseUrl}/api/v2/spans";
+        string zipkinEndpoint = $"http://host.docker.internal:{traceAgentPort}/api/v2/spans";
```

Comment:
Maybe the `host.docker.internal` prefix is causing the http listener to not accept the request?

Reply:
`host.docker.internal` seems busted for Windows containers, see https://github.com/docker/for-win/issues/1976 - I tried some of the scripts described on the issue but no luck so far.